runtime/gc_stats/ScavengerJavaStats.hpp

```diff
@@ -58,6 +58,9 @@ class MM_ScavengerJavaStats
 	MM_ReferenceStats _softReferenceStats;  /**< Soft reference stats for the cycle */
 	MM_ReferenceStats _phantomReferenceStats;  /**< Phantom reference stats for the cycle */
 
+	UDATA _monitorReferenceCleared; /**< The number of monitor references that have been cleared during marking */
+	UDATA _monitorReferenceCandidates; /**< The number of monitor references that have been visited in monitor table during marking */
+
```

Comment:
This piece of comment is "copied" from the definition of `_stringConstantsCleared` and repeated in several files. I realize this is only place '_monitorReferenceCleared' is defined but not `_stringConstantsCleared `. I assume you want to change from "marking" to  "during scavenge" because the difference in the nature of these operations. Do you mind explaining more?

Reply:
`ScavengerJavaStats` is specific for Scavenger (as the name implies) and Scavenger is a copying collector as opposed to a marking collector. The other stat classes are specific to marking collectors, which is why the comment is valid there.